src/Docker/Client/Types.hs

```diff
@@ -1422,4 +1423,3 @@ toJsonKeyVal vs getKey getVal = JSON.Object $ foldl f HM.empty vs
 -- | Helper function that return an empty dictionary "{}"
 emptyJsonObject :: JSON.Value
 emptyJsonObject = JSON.Object HM.empty
-
```

Comment:
This is style preference of mine. I like to keep an extra line at the end of each file. It's not relevant to this PR and I wont enforce it right now but we really should do a linter check of some sort...possibly even use some automated formatting so we don't have to do this manually. Anyway just a thought for the future. cc @jprider63